docs-src/concepts/what-is-advanced-visibility.md

```diff
@@ -9,9 +9,10 @@ tags:
   - visibility
 ---
 
-Advanced Visibility, within the Temporal Platform, is the subsystem and APIs that enable the listing, filtering, and sorting of Workflow Executions through a custom SQL-like [List Filter](/concepts/what-is-a-list-filter).
+Visibility, within the Temporal Platform, is the subsystem and APIs that enable the listing, filtering, and sorting of [Workflow Executions](/concepts/what-is-a-workflow-execution) through a custom SQL-like [List Filter](/concepts/what-is-a-list-filter).
 
-- In Temporal Cluster version 1.20 and later, advanced Visibility is available on SQL databases like MySQL (version 8.0.17 and later) and PostgreSQL (version 12 and later), in addition to support for Elasticsearch.
-- For Temporal Server versions 1.19.1 and earlier, you must [integrate with ElasticSearch](/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster) to use advanced Visibility.
-  Elasticsearch takes on the Visibility request load, relieving potential performance issues.
+- In Temporal Cluster version 1.20 and later, visibility is available on SQL databases like MySQL (version 8.0.17 and later) and PostgreSQL (version 12 and later), in addition to support for Elasticsearch.
+- For Temporal Server versions 1.19.1 and earlier, you must [integrate with ElasticSearch](/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster) to use visibility.
+  Elasticsearch takes on the visibility request load, relieving potential performance issues.
   We highly recommend operating a Temporal Cluster with Elasticsearch for any use case that spawns more than just a few Workflow Executions.
+- On Temporal Cloud, visibility is enabled by default for all users through the [Read-Only role](/cloud/how-to-get-started-with-temporal-cloud#invite-users)
```

Comment:
```suggestion
- On Temporal Cloud, advanced Visibility is enabled by default for all users through the [Read-Only role](/cloud/how-to-get-started-with-temporal-cloud#invite-users)
```

Reply:
Why is the `read-only` role important here? AFAIK this is enabled even if the account role is Developer and Global Admin combined with any of the Namespace-level permissions - please verify this with the cloud team.
Suggest reframing this to " On Temporal Cloud, advanced Visibility is enabled by default for all users."

Reply:
Yeah, it was tricky phrasing this. I'll go with your suggestion.